charts/core/src/Chart/Chart.types.ts

```diff
@@ -0,0 +1,65 @@
+import type { XAXisComponentOption, YAXisComponentOption } from 'echarts';
+import type { LineSeriesOption } from 'echarts/charts';
+import type {
+  DatasetComponentOption,
+  GridComponentOption,
+  LegendComponentOption,
+  TitleComponentOption,
+  ToolboxComponentOption,
+  TooltipComponentOption,
+} from 'echarts/components';
+import type { ComposeOption } from 'echarts/core';
+
+import { DarkModeProps, type HTMLElementProps } from '@leafygreen-ui/lib';
+
+type RequiredSeriesProps = 'type' | 'name' | 'data';
+export type SeriesOption = Pick<LineSeriesOption, RequiredSeriesProps> &
```

Comment:
Use PartialRequire

Reply:
Actually, leaving this for now. `PartialRequire<LineSeriesOption, RequiredSeriesProps>` doesn't seem to be typing this quite correctly, and I'm receiving errors when trying to access anything other than `type` of a `SeriesOption`.

Reply:
This _should_ work. Not sure why it's not